src/Form/FieldSelect.js

```diff
@@ -0,0 +1,153 @@
+import classNames from 'classnames';
```

Comment:
I think this is a good opportunity to use `classnames/dedupe`, but perhaps, for performance reasons, others may disagree.